book/src/configuration.md

```diff
@@ -65,6 +65,7 @@ Its settings will be merged with the configuration directory `config.toml` and t
 | `workspace-lsp-roots` | Directories relative to the workspace root that are treated as LSP roots. Should only be set in `.helix/config.toml` | `[]` |
 | `default-line-ending` | The line ending to use for new documents. Can be `native`, `lf`, `crlf`, `ff`, `cr` or `nel`. `native` uses the platform's native line ending (`crlf` on Windows, otherwise `lf`). | `native` |
 | `insert-final-newline` | Whether to automatically insert a trailing line-ending on write if missing | `true` |
+| `indent-heuristic` | How the indentation for a newly inserted line is computed: `simple` just copies the indentation level from the previous line, `tree-sitter` computes the indentation based on the syntax tree and `hybrid` combines both approaches | `hybrid`
```

Comment:
Is there a way to set a language setting for all languages at once? I think that the main use case of this option will be for debugging changes to the indent queries / heuristic, where changing the config for each relevant language might be tedious.

In case a language doesn't have queries, the `simple` heuristic is always used as a fallback (`hybrid` falls back to `tree-sitter` and `tree-sitter` falls back to `simple`). Buggy / low quality queries also shouldn't be a bug issue anymore. An empty indent query file with the `hybrid` heuristic should be essentially equivalent to using the `simple` heuristic and each added pattern will generally only improve the experience.
The only issue are incorrect patterns that e.g. often apply `@indent` where it isn't needed. Such patterns should never be added in the first place though, so I'm not sure if we need a mechanism to work around them.

Reply:
Ah that's even better than I missed that there was such a fallback on my first read. Now it's clearer that there is documentation for it 👍 

It is possible to create language settings with a global fallback but it does indeed seem like it's not needed here